src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,33 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
+
+@TSRFactory('herb', 'fuze_bottle', 'grasp')
+def fuze_grasp(robot, fuze, manip=None):
```

Comment:
I am thinking something like this:

```python
# Get the grasp tsr
grasp_tsr_list = robot.tsrlibrary(obj, 'grasp') 

# Add the standoff distance.
Tstandoff = numpy.array(
    [ 1., 0., 0., 0. ],
    [ 0., 1., 0., 0. ],
    [ 0., 0., 1., standoff_distance ],
    [ 0., 0., 0., 1. ],
])
for tsr_chain in grasp_tsr_list:
    tsr_chain.append(TSR(T0_w=Tstandoff, Tw_e=numpy.eye(4), Bw=numpy.zeros((6, 2)), manip=manip_idx))
```
(This is untested pseudo-code, so proceed with caution).